src/Cool/DebugOptions/generate_debug_options.py

```diff
@@ -70,6 +70,12 @@ def all_debug_options():
             kind=Kind.BUTTON,
             detailed_description="Allows you to click on any ImGui widget and have your IDE break on it, allowing you to find the source code that generated it.",
         ),
+        DebugOption(
```

Comment:
A name more in line with the other debug option names would be `Test "Did You Know?"`